### ISSUE TITLE
`pip install pyvkfft` fails when `nvcc` is available

### ISSUE THREAD

**Issue by inducer** [vincefn/pyvkfft#23]
On machines where `nvcc` is available, it seems that the source build for pyvkfft fails with a message like the following;

```
      /usr/bin/nvcc -I/usr/include -Isrc -c src/vkfft_cuda.cu -o build/temp.linux-x86_64-cpython-310/src/vkfft_cuda.o -O3 --ptxas-options=-v -std=c++11 --compiler-options=-fPIC
      cc1plus: fatal error: src/vkfft_cuda.cu: No such file or directory
```

See [here](https://gitlab.tiker.net/inducer/sumpy/-/jobs/513204) for a CI log that shows this in context. Note that the issue can be reproduced by just `pip install pyvkfft`. The machine where I am reproducing this has CUDA installed via Debian's repository, in case it matters. The CUDA version is 11.6.2.

**Comment by vincefn** [comment]
Strange. `pip install pyvkftt` (from pypi) works with just opencl, but indeed not with cuda. 
Local build using `pip install .` does work though.
Some path issue I guess (setuptools changed behaviour ?), I'll need a few days to get to it.

I deleted the pypi release in the meantime

**Comment by vincefn** [comment]
Ok, dumb mistake - `python setup.py sdist` did not pickup the `.cu` file, so the pypi package was incomplete. I had only tested the pypi install on my macbook so no cuda..

Should be fixed by 5800286cafb65283865e9d1621c9211f0087ebf8 and 2023.1.1 pypi package. Thanks !
